framework/tlvf/yaml/tlvf/WSC/WSC_Attributes.yaml

```diff
@@ -429,3 +429,254 @@ cWscVendorExtWfa:
   vs_data:
     _type: uint8_t
     _length: []
+
+cWscAttrVersion:
+  _type: class
+  _is_tlv_class: False
+  type:
+    _type: eWscAttributes
+    _value: ATTR_VERSION
+  length:
+    _type: uint16_t
+    _value: 1
+  data:
+    _type: eWscValues8
+    _value: WSC_VERSION
+
+cWscAttrMessageType:
+  _type: class
+  _is_tlv_class: False
+  type:
+    _type: eWscAttributes
+    _value: ATTR_MSG_TYPE
+  length:
+    _type: uint16_t
+    _value: 1
+  msg_type:
+    _type: eWscMessageType
+    _value: WSC_MSG_TYPE_INVALID
+
+cWscAttrEnroleeNonce:
+  _type: class
+  _is_tlv_class: False
+  type:
+    _type: eWscAttributes
+    _value: ATTR_ENROLLEE_NONCE
+  length:
+    _type: uint16_t
+    _value: WSC_NONCE_LENGTH
+  nonce:
+    _type: uint8_t
+    _length: [ WSC_NONCE_LENGTH ]
+
+cWscAttrPublicKey:
+  _type: class
+  _is_tlv_class: False
+  type:
+    _type: eWscAttributes
+    _value: ATTR_PUBLIC_KEY
+  length:
+    _type: uint16_t
+    _value: WSC_PUBLIC_KEY_LENGTH
+  public_key:
+    _type: uint8_t
+    _length: [ WSC_PUBLIC_KEY_LENGTH ]
+
+cWscAttrAuthenticationTypeFlags:
+  _type: class
+  _is_tlv_class: False
+  type:
+    _type: eWscAttributes
+    _value: ATTR_AUTH_TYPE_FLAGS
+  length: 
+    _type: uint16_t
+    _value: 2
+  auth_type_flags: 
+    _type: uint16_t
+    _value: uint16_t(eWscAuth::WSC_AUTH_OPEN) | uint16_t(eWscAuth::WSC_AUTH_WPA2PSK)
+
+cWscAttrEncryptionTypeFlags:
+  _type: class
+  _is_tlv_class: False
+  type:
+    _type: eWscAttributes
+    _value: ATTR_ENCR_TYPE_FLAGS
+  length:
+    _type: uint16_t
+    _value: 2
+  encr_type_flags:
+    _type: uint16_t
+    _value: uint16_t(eWscEncr::WSC_ENCR_NONE) | uint16_t(eWscEncr::WSC_ENCR_AES)
+  
+cWscAttrConnectionTypeFlags:
+  _type: class
+  _is_tlv_class: False
+  type:
+    _type: eWscAttributes
+    _value: ATTR_CONN_TYPE_FLAGS
+  length:
+    _type: uint16_t
+    _value: 1
+  conn_type_flags:
+    _type: eWscConn
+    _value: WSC_CONN_ESS
+  
+cWscAttrConfigurationMethods:
+  _type: class
+  _is_tlv_class: False
+  type:
+    _type: eWscAttributes
+    _value: ATTR_CONFIG_METHODS
+  length:
+    _type: uint16_t
+    _value: 2
+  conf_methods:
+    _type: uint16_t
+    _value: (WSC_CONFIG_PHY_PUSHBUTTON | WSC_CONFIG_VIRT_PUSHBUTTON)
+
+cWscAttrManufacturer:
+  _type: class
+  _is_tlv_class: False
+  type:
+    _type: eWscAttributes
+    _value: ATTR_MANUFACTURER
+  length:
+    _type: uint16_t
+    _length_var: True
+  manufacturer:
+    _type: char
+    _length: [ length ]
```

Comment:
My intuition would be to make this a dynamic length `[]` but I'm not sure if that makes much difference.